keyboards/idobo/config.h

```diff
@@ -20,11 +20,11 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
+#define VENDOR_ID       0x8010
```

Comment:
```suggestion
#define VENDOR_ID       0x6964 // ID
```

How about these?

Reply:
Any particular reason for this?

Reply:
It seems to be the general convention for VIA support:
https://caniusevia.com/docs/configuring_qmk/#change-vendor_id-product_id

Reply:
I can't figure out what am I missing. Isn't 0x8010 a valid Vendor ID value?